pandas/core/groupby/groupby.py

```diff
@@ -2898,14 +2907,16 @@ class Grouping(object):
     """
 
     def __init__(self, index, grouper=None, obj=None, name=None, level=None,
-                 sort=True, in_axis=False):
+                 sort=True, observed=None, in_axis=False):
```

Comment:
Why not have it as `False` default? (if we want to deprecate in the future, we can then just use `None` ?)